src/index.html

```diff
@@ -109,11 +113,11 @@ <h1>TOP 9 LARGEST COUNTRIES BY POPULATION</h1>
       </ol>
       <p class="total">
         Total:
-        <span class="total-population">Calculate it!</span>
+        <span class="total-population"></span>
       </p>
       <p class="average">
         Average:
-        <span class="average-population">Calculate it!</span>
+        <span class="average-population"></span>
       </p>
     </div>
     <script src="scripts/main.js"></script>
```

Comment:
This script tag duplicates the one in the head section. Consider removing it to avoid loading the same script twice, which can lead to performance issues.